lib/parser.ex

```diff
@@ -67,27 +56,21 @@ defmodule Membrane.Ogg.Parser do
         Enum.map(packets, fn packet ->
           %Packet{
             payload: packet,
-            track_id: bitstream_serial_number,
             bos?: (header_type &&& 0x2) > 0,
             eos?: (header_type &&& 0x4) > 0
           }
         end)
 
-      track_states =
-        Map.put(track_states, bitstream_serial_number, %{
-          continued_packet: new_incomplete_packet
-        })
-
-      {:ok, packets, track_states, rest}
+      {:ok, packets, new_incomplete_packet, rest}
     end
   end
 
   defp parse_header(
          <<"OggS", 0, header_type, _granule_position::little-signed-64,
-           bitstream_serial_number::little-unsigned-32, _page_sequence_number::32, _crc::32,
+           _bitstream_serial_number::little-unsigned-32, _page_sequence_number::32, _crc::32,
```

Comment:
we could raise if the serial number changes, since it would mean we have more than one stream (?)

Reply:
I think if we want such check it would be better to raise if demuxer received more than one OpusHead page